airflow/jobs/scheduler_job.py

```diff
@@ -49,6 +49,7 @@
 from airflow.ti_deps.dependencies_states import EXECUTION_STATES
 from airflow.utils import timezone
 from airflow.utils.callback_requests import DagCallbackRequest, TaskCallbackRequest
+from airflow.utils.cli import get_dag
```

Comment:
This function loads the actual DAG from the file on disk, which we can't do in the scheduler

Can't because Airflow shouldn't ever load DAGs in to any long running process which includes the scheduler. This is bad because it could mean a badly written dag file could bring down the scheduler (and right now the scheduler is isolated from this.